modules/collect/src/main/java/com/opengamma/strata/collect/io/CsvOutput.java

```diff
@@ -174,6 +174,218 @@ public static CsvOutput safe(Appendable underlying, String newLine, String separ
     return new CsvOutput(underlying, newLine, separator, true);
   }
 
+  /**
+   * Writes the provided {@code CsvFile} to the underlying, using the system default line separator and using a
+   * comma separator.
+   * <p>
+   * See the standard quoting rules in the class-level documentation.
+   *
+   * @param file  the file whose contents to output
+   * @param underlying  the destination to write to
+   */
+  public static void writeStandard(CsvFile file, Appendable underlying) {
```

Comment:
I think instance methods would be the better choice, as 12 methods should reduce to 2.

Reply:
Add an 'alwaysQuote' param to each (for consistency) and make that four?

Reply:
Actually, ignore that, safe/standard is already on the instance, isn't it.